setup.py

```diff
@@ -15,7 +15,10 @@
 from coalib.misc.BuildManPage import BuildManPage
 
 try:
-    locale.getlocale()
+    lc = locale.getlocale()
+    pf = platform.system()
+    if pf != 'Windows' and lc == (None, None):
```

Comment:
@Markman2 
I tested on both Linux platform and osx platform.
They worked fine.
but on windows, it looks like has different locale settings.
But I don't have windows machine.